templates/automotive/index.html

```diff
@@ -93,133 +101,103 @@ <h3 class="p-heading--2">Next level experience</h3>
       <div class="p-logo-section">
         <div class="p-logo-section__items u-no-margin--bottom">
           <div class="p-logo-section__item">
-            {{ image(url="https://assets.ubuntu.com/v1/85be122c-toyota-logo.png",
-                        alt="Toyota",
-                        width="288",
-                        height="288",
-                        hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-logo-section__logo"}) | safe
-            }}
+          {{ image(url="https://assets.ubuntu.com/v1/0db93697-toyota-logo.png",
+          alt="",
+          width="92",
+          height="157",
+          hi_def=True,
+          loading="auto|lazy"
+          ) | safe
+          }}
           </div>
           <div class="p-logo-section__item">
-            {{ image(url="https://assets.ubuntu.com/v1/e10b8957-vw-logo.png",
-                        alt="Volkswagen",
-                        width="288",
-                        height="288",
-                        hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-logo-section__logo"}) | safe
-            }}
+          {{ image(url="https://assets.ubuntu.com/v1/3840c0cb-vw-logo.png",
+          alt="",
+          width="81",
+          height="157",
+          hi_def=True,
+          loading="auto|lazy"
+          ) | safe
+          }}
           </div>
           <div class="p-logo-section__item">
-            {{ image(url="https://assets.ubuntu.com/v1/a3bb7ca4-mercedes-benz-group-logo.png",
-                        alt="",
-                        width="288",
-                        height="288",
-                        hi_def=True,
-                        loading="lazy",
-                        attrs={"class": "p-logo-section__logo"}) | safe
-            }}
+          {{ image(url="https://assets.ubuntu.com/v1/80aac625-mercedes-benz-group-logo.png",
+          alt="",
```

Comment:
```suggestion
          alt="Mercedez Benz",
```

Please add relevant alt texts for the all logos